client/internal/config.go

```diff
@@ -24,11 +38,17 @@ func createNewConfig(managementURL string, configPath string) (*Config, error) {
 	wgKey := generateKey()
 	config := &Config{PrivateKey: wgKey, WgIface: iface.WgInterfaceDefault, IFaceBlackList: []string{}}
 	if managementURL != "" {
-		config.ManagementURL = managementURL
+		URL, err := parseManagementURL(managementURL)
+		if err != nil {
+			return nil, err
+		}
+		config.ManagementURL = URL
 	} else {
-		config.ManagementURL = ManagementAddrDefault
+		config.ManagementURL = managementURLDefault
 	}
 
+	config.IFaceBlackList = []string{iface.WgInterfaceDefault, "tun0"}
```

Comment:
didn't get the reason for hardcoding blacklisting of tun0?

Reply:
Some users had issues when using OpenVPN alongside Wiretrustee. So I decided to exclude the default one. 
I think we could do a better job of discovering the interfaces we want for ICE without hardcoding but for now, should be fine.

Reply:
Got it. thanks